BUILDING.md

```diff
@@ -88,6 +88,23 @@ Depending on host platform, the selection of toolchains may vary.
 
 * Visual Studio 2017 or the Build Tools thereof
 
+#### OpenSSL asm support
+
+OpenSSL-1.1.0 requires the following asssember version for use of asm
```

Comment:
`asssember` -> `assembler`

Reply:
Fixed in 07d5ac52ee66ada4e4a716edb0d4880360b122d6.